pages/linux/xbps.md

```diff
@@ -0,0 +1,23 @@
+# xbps
+
+> The X Binary Package System (or xbps) is the binary package system used by Void Linux.
+
+- Install packages:
+
+`xbps-install -S {{package_name1}} {{package_name2}}`
```

Comment:
Same here. `-S` doesn't match with 'install'/

Reply:
"The -S, --sync flag can be used while installing or updating to be always in sync with remote repositories"

As you can see -s and -S are different.

The whole thing is quite complicated and I think of adding few other pages (xbps-install, xbps-query, xbps-remove etc.) But those few are the most used ones, and if I put more information in the tldr page I will contradict the purpose and the guidelines of tldr pages. :smiley:

Reply:
Yea, this does seem difficult.